src/send_kcidb.py

```diff
@@ -166,6 +166,15 @@ def _get_log_excerpt(self, log_url):
             return data[-(16*1024):]
 
     def _parse_build_node(self, origin, node):
+        result = node.get('result')
+        result_map = {
+            'pass': True,
+            'fail': False,
+            'incomplete': None,
```

Comment:
nit: `incomplete` would get `valid=None` even if it was absent in the map - no pressure to remove it though

Reply:
I kept it as KCIDB will have a different status for it at some point and then we can update this mapping.